workflow/react-project/src/components/generic/Modal/style.less

```diff
@@ -0,0 +1,74 @@
+@import "../../../styles/fonts.less";
+@import "../../../styles/colors.less";
+@import "../../../styles/mixins.less";
+
+.dark-background {
+  position: fixed;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  z-index: 1;
```

Comment:
Header has z-index more, so when Random Cocktail modal is opened we have ability to open Authorisation modal, it's not good